lib/views/app.js

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
App logic has been moving into an `App` view. This means the list is now a dumb(er) view.